v3.0/reference/sql/statements/load-data.md

```diff
@@ -49,4 +49,3 @@ Records: 815264  Deleted: 0  Skipped: 0  Warnings: 0
 
 * [INSERT](/reference/sql/statements/insert.md)
 * [Transaction Model](/reference/transactions/transaction-model.md)
-* [Import Example Database](/how-to/get-started/import-example-database.md)
```

Comment:
I can't find where it is. Was it deleted already?

Reply:
@cofyc Yes. We only keep this doc in the English version. Just delete this line.